docs/.vuepress/theme/utils/translations.js

```diff
@@ -363,6 +363,96 @@ const translations = {
     'page-learn': '배우기',
     'page-developers': '개발자 가이드'
   },
+  'nl-NL': {
+    path: '/nl/',
+    name: 'Nederlands',
+    'english-name': 'Dutch',
+    ethereum: 'Ethereum',
+    'page-home': 'Ethereum',
+    'page-home-title': 'Ethereum is een wereldwijd, open-source platform voor gedecentraliseerde toepassingen.',
+    'page-home-subtitle': 'Op Ethereum kunt u code schrijven die digitale waarde beheert, precies werkt zoals het is geprogrammeerd, en overal ter wereld toegankelijk is.',
+    'page-home-section-beginners-title': 'Beginners',
+    'page-home-section-beginners-item-one': 'Nieuw bij Ethereum?',
+    'page-home-section-beginners-item-two': 'Wat is Ethereum?',
+    'page-home-section-beginners-item-three': 'Wat heb ik eraan?',
+    'page-home-section-use-title': 'Gebruik',
+    'page-home-section-use-item-one': 'Wat kan ik nu doen met Ethereum?',
+    'page-home-section-use-item-one-link': '/use/#_1-use-an-application-built-on-ethereum',
+    'page-home-section-use-item-two': 'Hoe krijg ik Ether?',
+    'page-home-section-use-item-two-link': '/use/#_2-what-is-eth-and-how-do-i-get-it',
+    'page-home-section-use-item-three': 'Wat is een \'wallet\' of digitale portemonnee?',
+    'page-home-section-use-item-three-link': '/use/#_3-what-is-a-wallet-and-which-one-should-i-use',
+    'page-home-section-learn-title': 'Info',
+    'page-home-section-learn-item-one': 'Beginnersinformatie',
+    'page-home-section-learn-item-one-link': '/learn/#ethereum-basics',
+    'page-home-section-learn-item-two': 'Hoe Ethereum werkt',
+    'page-home-section-learn-item-two-link': '/learn/#how-ethereum-works',
+    'page-home-section-learn-item-three': 'ETH 2.0',
+    'page-home-section-learn-item-three-link': '/learn/#eth-2-0',
+    'page-home-section-developers-title': 'Ontwikkelaars',
+    'page-home-section-developers-item-one': 'Snel aan de slag',
+    'page-home-section-developers-item-one-link': '/developers/#getting-started',
+    'page-home-section-developers-item-two': 'Leer slimme contracten te programmeren',
+    'page-home-section-developers-item-two-link': '/developers/#smart-contract-languages',
+    'page-home-section-developers-item-three': 'Vind de nieuwste tools voor ontwikkelaars',
+    'page-home-section-developers-item-three-link': '/developers/#developer-tools',
+    'page-home-section-enterprise-title': '',
+    'page-home-section-enterprise-item-one': '',
+    'page-home-section-enterprise-item-one-link': '',
+    'page-home-section-enterprise-item-two': '',
+    'page-home-section-enterprise-item-two-link': '',
+    'page-home-section-enterprise-item-three': '',
+    'page-home-section-enterprise-item-three-link': '',    
+    'page-beginners': 'Beginners',
+    'page-use': 'Gebruik',
+    'page-learn': 'Info',
+    'page-developers': 'Ontwikkelaars',
+  },
+  'pl-PL': {
```

Comment:
Same deal here - please update this key to match CrowdIn (`pl` - https://crowdin.com/project/ethereumfoundation/pl)